### PR TITLE
tiup: add doc for maintaining a tidb cluster using tiup

### PR DESCRIPTION
**Pull request by ran-huang**
### What is changed, added or deleted? (Required)

Add doc for maintaining a TiDB cluster using TiUP.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

**If you select two or more versions from above**, to trigger the bot to cherry-pick this PR to your desired release version branch(es), you **must** add corresponding labels such as **needs-cherry-pick-4.0**, **needs-cherry-pick-3.1**, **needs-cherry-pick-3.0**, and **needs-cherry-pick-2.1**.

### What is the related PR or file link(s)?

- This PR is translated from:https://github.com/pingcap/docs-cn/pull/2816
- Other reference link(s):

### CODE REVIEW

File: how-to/maintain/tiup-operations.md [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+
+If you want to set the transaction size limit parameter (`txn-total-size-limit` in the [performance](https://github.com/pingcap/tidb/blob/v4.0.0-rc/config/config.toml.example) module) to `1G` in tidb-server, edit the configuration as follows:
+
+```
+server_configs:
+  tidb:
+    performance.txn-total-size-limit: 1073741824
+```
+
+Run the `tiup cluster reload ${cluster-name} -N tidb` command to rolling restart the TiDB component.
```
Comment (yikeke) [inline how-to/maintain/tiup-operations.md:135]:
```suggestion
Then run the `tiup cluster reload ${cluster-name} -N tidb` command to rolling restart the TiDB component.
```

File: how-to/maintain/tiup-operations.md [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+
+## Stop the cluster
+
+The components in the TiDB cluster are stopped in the following order (The monitoring component is also stopped):
+
+Drainer -> TiFlash -> TiDB -> Pump -> TiKV -> PD
```
Comment (yikeke) [inline how-to/maintain/tiup-operations.md:141]:
```suggestion
**Drainer -> TiFlash -> TiDB -> Pump -> TiKV -> PD**
```

**Comment by ran-huang** [comment]
@yikeke Comments addressed. PTAL~

**Pull request merged**